ci-operator/step-registry/interop-tests/deploy-odf/interop-tests-deploy-odf-ref.yaml

```diff
@@ -18,7 +18,7 @@ ref:
     Deploy ODF
   env:
   - name: ODF_VERSION_MAJOR_MINOR
-    documentation: Openshift major.minor version
+    documentation: The odf major.minor version
     default: "4.13"
```

Comment:
So does this scenario test ODF 4.13 with OCP 4.16? At some point I remember that ODF was aiming to maintain parity with OCP versions? Is that not the case anymore?

Reply:
@btjd Thanks for the review

1. The ODF operator recently released version 4.15, which I use for testing and modify both in my OCP 4.15 and 4.16 configs. See [REF](https://github.com/openshift/release/pull/48201/files#diff-62ee4e9aad09b895797e2d554603fc3781a77e6f214c35c5c6b3f76c08e9033dR57)

2. The `ODF_VERSION_MAJOR_MINOR` env var documentation wrongfully mentions Openshift version instead of ODF, I updated it, since it caused problems.
Moved this description update to a separate PR
https://github.com/openshift/release/pull/50422

Reply:
@btjd Also, since many jobs are using this ref, I didn't want to update the ODF version default value ("4.13"), but to set it on my configs.